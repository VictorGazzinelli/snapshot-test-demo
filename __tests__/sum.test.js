const sum = require('../sum');

test('snapshot test for sum function', () => {
    expect(sum(1, 2)).toMatchSnapshot();
});

test('other snapshot test for sum function', () => {
    expect(sum(3, 4)).toMatchSnapshot();
});
