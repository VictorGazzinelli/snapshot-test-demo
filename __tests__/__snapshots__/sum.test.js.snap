// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`other snapshot test for sum function 1`] = `7`;

exports[`snapshot test for sum function 1`] = `3`;
